Scripts/cmake/Platform/iOS/Toolchain_ios.cmake

```diff
@@ -18,3 +18,7 @@ set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LIBRARY "libc++")
 
 set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE NO)
 
+set(CMAKE_XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET "14.0" CACHE STRING "The minimum IOS Version to support" FORCE)
```

Comment:
This should be iOS 13.0. 

Updating all of O3DE to require iOS14 is possible, but would need wider discussion.

Reply:
I guess the min iOS revision was updated to 14 and I didn't notice, please ignore this comment :)

Reply:
The Toolchain_ios.cmake for the engine was also set to iOS 14 last week. I can change it back but what are the issues with setting it to 14? Since iOS 15 came out recently, I doubt many devices would still be on iOS 13.